Python/Source/Arithmetics.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Arithmetics.hpp"
+
+namespace Rover::Details {
+  const pybind11::module& import_operator() {
+    static auto module = pybind11::module::import("operator");
+    return module;
+  }
+}
+
+namespace pybind11 {
+  object abs(const object& obj) {
+    return module::import("builtins").attr("abs")(obj);
```

Comment:
Similar to arithmetic, write a function to import the module you need and reuse that import to avoid constantly importing the module on every invocation.

Reply:
Sorry forgot this one, will commit shortly.